src/notebook/index.css

```diff
@@ -9,6 +9,11 @@
 }
 
 
+.p-Widget * + p { 
```

Comment:
This is a pretty broad selector (and expensive to boot). What are wanting to accomplish here? There might be a better way to select it, or we can add an appropriate class name to make it easier to target.